hv-rhel6.x/hv/vmbus_drv.c

```diff
@@ -1339,8 +1369,12 @@ static int __init hv_acpi_init(void)
 static void __exit vmbus_exit(void)
 {
 	int cpu;
+#if (RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(7,2))
+	hv_remove_kexec_handler();
+	hv_remove_crash_handler();
+#endif
 	vmbus_connection.conn_state = DISCONNECTED;
-	hv_synic_clockevents_cleanup();
+//      hv_synic_clockevents_cleanup();  will comment this for time being till clockevents_unbind showed up in distro code
```

Comment:
I think the following would be more appropriate:
#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(6,8))
        hv_synic_clockevents_cleanup();
#endif

Wrapping the code with the #if statement, if the code shows up in 6.9, it will just work.  If not, it will fail to compile on 6.9, but not be an issue on the older releases.